main.go

```diff
@@ -66,10 +70,15 @@ BE ON THE LOOKOUT !
 		for {
 
 			var input string
-			if _, err := fmt.Scanf("%s\n", &input); err != nil {
 
+			char, key, err := keyboard.GetSingleKey()
+			if err != nil {
 				log.Println(err)
 			}
+			defer fmt.Printf("snap: %q\r\n", char)
+			if key == keyboard.KeyEsc {
+				break
+			}
 			inputChannel <- input
```

Comment:
input is always an empty string. It would be more expected to put the char or key into the input channel